modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -73,6 +80,53 @@ const AddFDSFieldsModalContent = ({
 		);
 	};
 
+	const synchronizeFDSFields = async () => {
+		const creationData: Array<{name: string; type: string}> = [];
+		const deletionIds: Array<number> = [];
+
+		fields?.forEach((field) => {
+			if (field.selected && !field.id) {
+				creationData.push({name: field.name, type: field.type});
+			}
+
+			if (!field.selected && field.id) {
+				deletionIds.push(field.id);
+			}
+		});
+
+		const formData = new FormData();
+
+		formData.append(
+			`${namespace}creationData`,
+			JSON.stringify(creationData)
+		);
+
+		deletionIds.forEach((id) => {
+			formData.append(`${namespace}deletionIds`, String(id));
+		});
+
+		formData.append(`${namespace}fdsViewId`, fdsView.id);
```

Comment:
Is FormData necessary? Or can we just pass these as stringified json? Only think this because our other cases of POSTing to FDSView we use json data

Reply:
It might be necessary. I initially unsuccessful tried to do it, but admittedly I didn't try too hard. It depends on what resource request API allows. `FormData` is the typical pattern in DXP with resource requests, while JSON is common with REST endpoints. I can try digging a bit more.

Reply:
Ah gotcha, that makes sense. No worries if JSON doesn't work, I mainly was just thinking of how to keep it consistent.